### PR TITLE
Merge changes from 19.4.0.1 beta

### PR DESCRIPTION
**Pull request by mokagio**
Includes:

- Update the default stats cards list [(#18072)](https://github.com/wordpress-mobile/WordPress-iOS/pull/18072) by @Gio2018 
- Show the FAB button when creating a site, hide it when deleting one [(#18110)](https://github.com/wordpress-mobile/WordPress-iOS/pull/18110) by @leandroalonso 

Nothing to test, if CI Shows green, we're good. **Note that the UI tests failure on iPad is expect and is being addressed in #18104.**

## Regression Notes
1. Potential unintended areas of impact
N.A.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N.A.

3. What automated tests I added (or what prevented me from doing so)
N.A.

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -6,8 +6,9 @@
 -----
 * [*] Site Creation: Fixed layout of domain input field for RTL languages. [#18006]
 * [*] [internal] The FAB (blue button to create posts/stories/pages) creation/life cycle was changed [#18026]
-* [*] Push notifications will now display rich media when long pressed. [#18048]
 * [*] Stats: we fixed a variety of performance issues in the Insight screen. [#17926, #17936, #18017]
+* [*] Stats: we re-organized the default view in Insights, presenting more interesting data at a glance [#18072]
```
Comment (AliSoftware) [inline RELEASE-NOTES.txt:10]: Since this is a new entry, do you want to try to add mention of it to Grace's editorialized notes, or is it too late?

**Review by AliSoftware — APPROVED** [review]
✅ Contains changes from the beta fixes which landed in `release/*` branch
✅  Latest `.strings` translations pulled from GlotPress
✅  `release_notes.txt` translated files emptied for the new version, in preparation for getting the new notes later in the sprint.
✅ Version bump in `.xcconfig`

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (mokagio) [reply RELEASE-NOTES.txt:10]: For reference, see https://github.com/wordpress-mobile/WordPress-iOS/pull/18116#discussion_r824296549:

> I think it's fine to keep [the release notes] like this, but thank you for pointing them out.

**Pull request merged**